fire/inspectutils.py

```diff
@@ -63,46 +77,40 @@ def GetArgSpec(fn):
   Args:
     fn: The function to analyze.
   Returns:
-    A named tuple of type inspect.ArgSpec with the following fields:
+    An object containing the following fields:
       args: A list of the argument names accepted by the function.
```

Comment:
Let's move these arg descriptions to the docstring of _FullArgSpec's init, and then the docstring here can be much simpler, something like:

```
...
Returns:
  A FullArgSpec object for the provided function.
```